### ISSUE TITLE
Don't complain about unused methods that override a vendor one

### ISSUE THREAD

**Issue by muglug** [vimeo/psalm#910]
Given

```php
class VendorClass {
  public function foo() : void {}
}
class MyClass extends VendorClass {
  public function foo() : void {}
}
```

Psalm should not complain about unused methods in `MyClass` that override (or implement) same-named methods ins `VendorClass`
